<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="btnLayout">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="prevNextStudentLayout">
      <item>
       <widget class="QPushButton" name="prevStudentBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous Student</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextStudentBtn">
        <property name="text">
         <string>Next Student</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="lowerControlsLayout">
      <item>
       <widget class="QPushButton" name="finishBtn">
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="commentBox"/>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="prevNextStandardLayout">
      <item>
       <widget class="QPushButton" name="prevStandardBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous Standard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextStandardBtn">
        <property name="text">
         <string>Next Standard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="gradeTab">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="kTab">
       <attribute name="title">
        <string>K</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="kStandardsDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="kStudentDropdown"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="firstTab">
       <attribute name="title">
        <string>1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QComboBox" name="firstStandardDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="firstStudentDropdown"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="secondTab">
       <attribute name="title">
        <string>2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QComboBox" name="secondStandardDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="secondStudentDropdown"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="thirdTab">
       <attribute name="title">
        <string>3</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QComboBox" name="thirdStandardDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="thirdStudentDropdown"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fourthTab">
       <attribute name="title">
        <string>4</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QComboBox" name="fourthStandardDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="fourthStudentDropdown"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fifthTab">
       <attribute name="title">
        <string>5</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QComboBox" name="fifthStandardDropdown"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="fifthStudentDropdown"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="ratingLayout">
      <item row="0" column="3">
       <widget class="QRadioButton" name="threeBtn">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="oneBtn">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="zeroBtn">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="fourBtn">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="twoBtn">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="zeroLineEdit">
        <property name="text">
         <string>No data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="oneLineEdit">
        <property name="text">
         <string>Student has been introduced to this standard and is unable to meet standard at this time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="twoLineEdit">
        <property name="text">
         <string>Student is showing progress towards standard, but is still struggling at times</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="threeLineEdit">
        <property name="text">
         <string>Student has completed standard</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="fourLineEdit">
        <property name="text">
         <string>Student is excelling at this standard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
